src/logger.cpp

```diff
@@ -462,8 +465,8 @@ void Logger::SetLogFacility(LogFacility facility, std::string filename)
         // Open syslog if appropriate
         if (facility == LogFacility::Syslog)
         {
-#ifdef _WIN32
-            error << "Syslog not supported on Windows" << std::flush;
+#ifndef LOGGER_SYSLOG_ENABLED
+            error << "Syslog is not supported on this platform" << std::flush;
 #else
```

Comment:
Should this throw an error? Or return early?

Reply:
Just return early. Worst case is logging will go to the console if not observed. This isn't catastrophic.